### ISSUE TITLE
Cannot invoke "me.neznamy.tab.bridge.bukkit.BridgePlayer.sendPacket(Object)"

### ISSUE THREAD

**Issue by james090500** [NEZNAMY/TAB#522]
### Server version

Velocity 3.1.2 - Paper 1.18.2 225

### TAB version

TAB 3.10 Velociy - Tab Bridge 2.0.0

### Stack trace

If a player joins the server just after it started they recieve this error. 

```
[22:45:11 WARN]: java.lang.NullPointerException: Cannot invoke "me.neznamy.tab.bridge.bukkit.BridgePlayer.sendPacket(Object)" because the return value of "me.neznamy.tab.bridge.bukkit.BukkitBridge.getPlayer(org.bukkit.entity.Player)" is null
[22:45:11 WARN]:        at TAB-Bridge v2.0.0.jar//me.neznamy.tab.bridge.bukkit.DataBridge.processPluginMessage(DataBridge.java:180)
[22:45:11 WARN]:        at TAB-Bridge v2.0.0.jar//me.neznamy.tab.bridge.bukkit.BukkitBridge.lambda$onPluginMessageReceived$0(BukkitBridge.java:88)
[22:45:11 WARN]:        at TAB-Bridge v2.0.0.jar//me.neznamy.tab.bridge.bukkit.DataBridge.lambda$submitTask$4(DataBridge.java:193)
[22:45:11 WARN]:        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[22:45:11 WARN]:        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[22:45:11 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[22:45:11 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[22:45:11 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833)
[22:45:11 WARN]: java.lang.NullPointerException: Cannot invoke "me.neznamy.tab.bridge.bukkit.BridgePlayer.sendPacket(Object)" because the return value of "me.neznamy.tab.bridge.bukkit.BukkitBridge.getPlayer(org.bukkit.entity.Player)" is null
[22:45:11 WARN]:        at TAB-Bridge v2.0.0.jar//me.neznamy.tab.bridge.bukkit.DataBridge.processPluginMessage(DataBridge.java:151)
[22:45:11 WARN]:        at TAB-Bridge v2.0.0.jar//me.neznamy.tab.bridge.bukkit.BukkitBridge.lambda$onPluginMessageReceived$0(BukkitBridge.java:88)
[22:45:11 WARN]:        at TAB-Bridge v2.0.0.jar//me.neznamy.tab.bridge.bukkit.DataBridge.lambda$submitTask$4(DataBridge.java:193)
[22:45:11 WARN]:        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[22:45:11 WARN]:        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[22:45:11 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[22:45:11 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[22:45:11 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833)
```

### Steps to reproduce (if known)

Step 1. Restart server
Step 2. Join as soon as you can

### Checklist

- [X] I am running latest version of the plugin
- [X] I have included a paste of the error
- [X] I have read the Compatibility wiki page and am not trying to run the plugin on an unsupported server version / platform

**Comment by james090500** [comment]
I realise this is probably a TAB Bridge issue. I can move to https://github.com/NEZNAMY/TAB-Bridge if needed?

**Comment by NEZNAMY** [comment]
It depends. There's a chance the proxy TAB sent plugin messages in wrong order. Doesn't make much sense otherwise. Is this on join or server switch?

**Comment by NEZNAMY** [comment]
Managed to reproduce it, although it's extremely rare for me. Try https://github.com/NEZNAMY/TAB-Bridge/actions/runs/1954121204.

**Comment by james090500** [comment]
That's sorted. Thanks!

**Comment by james090500** [comment]
I'm affraid this happened again:

```
[19:55:16 INFO]: [LuckPerms] [Messaging] Sending user ping for 'james090500' with id: cbd19d6f-7137-4400-bb09-999099f20954
[19:55:16 WARN]: java.lang.NullPointerException: Cannot invoke "me.neznamy.tab.bridge.bukkit.features.unlimitedtags.ArmorStandManager.getArmorStand(String)" because the return value of "me.neznamy.tab.bridge.bukkit.features.unlimitedtags.BridgeNameTagX.getArmorStandManager(me.neznamy.tab.bridge.bukkit.BridgePlayer)" is null
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.features.unlimitedtags.BridgeNameTagX.readMessage(BridgeNameTagX.java:219)
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.DataBridge.processPluginMessage(DataBridge.java:125)
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.BukkitBridge.lambda$onPluginMessageReceived$0(BukkitBridge.java:88)
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.DataBridge.lambda$submitTask$8(DataBridge.java:214)
[19:55:16 WARN]:        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[19:55:16 WARN]:        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[19:55:16 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[19:55:16 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[19:55:16 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833)
[19:55:16 WARN]: java.lang.NullPointerException: Cannot invoke "me.neznamy.tab.bridge.bukkit.features.unlimitedtags.ArmorStandManager.getArmorStand(String)" because the return value of "me.neznamy.tab.bridge.bukkit.features.unlimitedtags.BridgeNameTagX.getArmorStandManager(me.neznamy.tab.bridge.bukkit.BridgePlayer)" is null
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.features.unlimitedtags.BridgeNameTagX.readMessage(BridgeNameTagX.java:219)
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.DataBridge.processPluginMessage(DataBridge.java:125)
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.BukkitBridge.lambda$onPluginMessageReceived$0(BukkitBridge.java:88)
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.DataBridge.lambda$submitTask$8(DataBridge.java:214)
[19:55:16 WARN]:        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[19:55:16 WARN]:        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[19:55:16 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[19:55:16 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[19:55:16 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833)
[19:55:16 WARN]: java.lang.NullPointerException: Cannot invoke "me.neznamy.tab.bridge.bukkit.features.unlimitedtags.ArmorStandManager.getArmorStand(String)" because the return value of "me.neznamy.tab.bridge.bukkit.features.unlimitedtags.BridgeNameTagX.getArmorStandManager(me.neznamy.tab.bridge.bukkit.BridgePlayer)" is null
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.features.unlimitedtags.BridgeNameTagX.readMessage(BridgeNameTagX.java:219)
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.DataBridge.processPluginMessage(DataBridge.java:125)
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.BukkitBridge.lambda$onPluginMessageReceived$0(BukkitBridge.java:88)
[19:55:16 WARN]:        at TAB-Bridge-2.0.0.jar//me.neznamy.tab.bridge.bukkit.DataBridge.lambda$submitTask$8(DataBridge.java:214)
[19:55:16 WARN]:        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[19:55:16 WARN]:        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[19:55:16 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[19:55:16 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[19:55:16 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833)
[19:55:16 INFO]: james090500[/145.224.67.14:52732] logged in with entity id 1 at ([world]-2.5, 65.0, -5.5)
[19:55:17 WARN]: [HolographicDisplays] Couldn't fetch data from survival (172.18.0.1:27002).
java.net.SocketException: Connection reset
        at sun.nio.ch.NioSocketImpl.implRead(NioSocketImpl.java:323) ~[?:?]
        at sun.nio.ch.NioSocketImpl.read(NioSocketImpl.java:350) ~[?:?]
        at sun.nio.ch.NioSocketImpl$1.read(NioSocketImpl.java:803) ~[?:?]
        at java.net.Socket$SocketInputStream.read(Socket.java:966) ~[?:?]
        at java.net.Socket$SocketInputStream.read(Socket.java:961) ~[?:?]
        at java.io.DataInputStream.readByte(DataInputStream.java:271) ~[?:?]
        at me.filoghost.holographicdisplays.plugin.lib.fcommons.ping.MinecraftServerPinger.readVarInt(MinecraftServerPinger.java:75) ~[HolographicDisplays-3.0.0-SNAPSHOT.jar:?]
        at me.filoghost.holographicdisplays.plugin.lib.fcommons.ping.MinecraftServerPinger.ping(MinecraftServerPinger.java:37) ~[HolographicDisplays-3.0.0-SNAPSHOT.jar:?]
        at me.filoghost.holographicdisplays.plugin.bridge.bungeecord.BungeeServerTracker.pingServer(BungeeServerTracker.java:103) ~[HolographicDisplays-3.0.0-SNAPSHOT.jar:?]
        at me.filoghost.holographicdisplays.plugin.bridge.bungeecord.BungeeServerTracker.updateServerInfoWithPinger(BungeeServerTracker.java:88) ~[HolographicDisplays-3.0.0-SNAPSHOT.jar:?]
        at me.filoghost.holographicdisplays.plugin.bridge.bungeecord.BungeeServerTracker.lambda$runPeriodicUpdateTask$1(BungeeServerTracker.java:74) ~[HolographicDisplays-3.0.0-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftTask.run(CraftTask.java:101) ~[paper-1.18.2.jar:git-Paper-255]
        at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57) ~[paper-1.18.2.jar:git-Paper-255]
        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[paper-1.18.2.jar:git-Paper-255]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
[19:55:30 INFO]: [H] RESPECTED » James: Test
```

**Comment by NEZNAMY** [comment]
That's a different error though.

**Comment by james090500** [comment]
I apologise. would you like me to make a new issue?

**Comment by NEZNAMY** [comment]
Doesn't really matter.

**Comment by NEZNAMY** [comment]
@james090500 https://github.com/NEZNAMY/TAB-Bridge/actions/runs/2024110034

**Comment by NEZNAMY** [comment]
Fixed in 2.0.1
